components/nexus-blobstore-api/pom.xml

```diff
@@ -35,9 +35,32 @@
 
     <dependency>
       <groupId>org.sonatype.nexus</groupId>
+      <artifactId>nexus-orient</artifactId>
```

Comment:
api module should not depend on orient, move the orient impl to the nexus-core internal package for blobstore.